azure-pipelines.yml

```diff
@@ -168,6 +168,8 @@ stages:
             value: $(AutoTestClientId_NoAuth)
           - name: "EssAuthorizationConfiguration.AutoTestClientSecretNoAuth"
             value: $(AutoTestClientSecret_NoAuth)
+          - name: "SalesCatalogue.BaseUrl"
+            value: "https://m-salescatstub-api-dev-appservice.azurewebsites.net"
```

Comment:
Would updating the variable group not be a better idea than hard coding our configuration?

Reply:
My initial thought was to put it on library then I thought since they are not secrets I added it on yml but make sense if we keep it on Library won't have to go through PR if we want to change the urls. I will remove them from here.